gateway/crates/config/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+edition.workspace = true
+homepage.workspace = true
+keywords.workspace = true
+license = "MPL-2.0"
+name = "gateway-config"
+repository.workspace = true
+version = "0.6.1"
+
+[lints]
+workspace = true
+
+[dependencies]
+ascii = { version = "1.1.0", features = ["serde"] }
+duration-str = "0.11.0"
+engine.workspace = true
+grafbase-telemetry = { workspace = true, features = ["tower", "otlp"] }
```

Comment:
I'm not really sure about this. Should we actually just have the configuration options for telemetry in the config crate, so we don't get this kind of crazy dependency chain here...